### DISCUSSION TITLE
How to execute command line automatically when start tabby?

### DISCUSSION THREAD

**Discussion by hustc12** [Eugeny/tabby#9319] [Q&A]
My case is that I need to activate the conda (base) environment when starts Tabby on Windows. The command line is "conda activate base". How could I enable it when starting Tabby?

**Comment by AlexGT** [comment]
Add your command to ~/.bash_profile ?

**Reply by hustc12** [reply]
Thank you for your advice! I am not sure whether it could work due to I am running on Windows. Anyway, I will have a try. Thanks again!
